### DISCUSSION TITLE
Last verified as a null

### DISCUSSION THREAD

**Discussion by cjayo** [community/community#110645] [GitHub Education]
### Hi everyone,

Hello, I have applied for the student benefits on the 26th but it is saying that I was verified as a null. It's been more than three days since i was approved but I have not recieved a email so I was wondering if it is a glitch or if i should wait longer to access the benefits
![Screenshot 2024-03-01 174535](https://github.com/community/community/assets/146975903/422a30e0-b163-4827-a4db-6dcfb5244ac7)

**Comment by cjayo** [comment]
UPDATE: i got it thanks guys

**Reply by GCustom** [reply]
did you have to do anything?

**Reply by cjayo** [reply]
just had to wait a bit longer, but if it is taking a while def make a post about it so they know whats happening
